capture-sdk/sdk/src/main/java/net/gini/android/capture/tracking/useranalytics/UserAnalyticsScreen.kt

```diff
@@ -0,0 +1,41 @@
+package net.gini.android.capture.tracking.useranalytics
+
+import net.gini.android.capture.R
+
+enum class UserAnalyticsScreen(val screenName: String) {
+    CAMERA("camera"),
+    REVIEW("review"),
+    REVIEW_ZOOM("review_zoom"),
+    ANALYSIS("analysis"),
+    NO_RESULTS("no_results"),
+    HELP("help"),
+    ERROR("error"),
+    ONBOARDING_FLAT_PAPER("onboarding_flat_paper"),
+    ONBOARDING_LIGHTING("onboarding_lighting"),
+    ONBOARDING_MULTIPLE_PAGES("onboarding_multiple_pages"),
+    ONBOARDING_QR_CODE("onboarding_qr_code"),
+    ONBOARDING_CUSTOM_1("onboarding_custom_1"),
+    ONBOARDING_CUSTOM_2("onboarding_custom_2"),
+    ONBOARDING_CUSTOM_3("onboarding_custom_3"),
+    ONBOARDING_CUSTOM_4("onboarding_custom_4"),
+    ONBOARDING_CUSTOM_5("onboarding_custom_5"),
```

Comment:
[Request]
We need to refactor this part because there is can be more than 5 custom pages in onboarding 
(Clarified this with Valentina)